packages/components/package.json

```diff
@@ -29,7 +29,9 @@
         "react-dom": ">=16.8",
         "react-native": ">=0.68",
         "expo-font": ">=10.0.0",
-        "react-native-svg": "*"
+        "react-native-svg": "*",
```

Comment:
Will this package work with all of the 13 different major versions of react-native-svg?

Reply:
Technically, no. I should perhaps force the major version at least

Reply:
fixed in db03b57